src/openrpc/discovery.json

```diff
@@ -52,6 +52,10 @@
         {
           "name": "capabilities",
           "x-provides": "xrn:firebolt:capability:discovery:entity-info"
+        },
+        {
+          "name": "deprecated",
```

Comment:
I'm leaning towards deprecating them anyway, since we know we're going to redesign them. Want app developer to know the APIs aren't going to stick around.